PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -3003,6 +3003,50 @@ void ScintillaEditView::performGlobalStyles()
 	execute(SCI_SETMARGINTYPEN, _SC_MARGE_CHANGEHISTORY, SC_MARGIN_COLOUR);
 	execute(SCI_SETMARGINBACKN, _SC_MARGE_CHANGEHISTORY, changeHistoryMarginColor);
 
+	COLORREF changeModifiedfgColor = orange;
+	pStyle = stylers.findByName(TEXT("Change History modified"));
+	if (pStyle)
+	{
+		changeModifiedfgColor = pStyle->_fgColor;
+	}
+	execute(SCI_MARKERSETFORE, SC_MARKNUM_HISTORY_MODIFIED, changeModifiedfgColor);
+	execute(SCI_MARKERSETBACK, SC_MARKNUM_HISTORY_MODIFIED, changeModifiedfgColor);
+	execute(SCI_INDICSETFORE, INDICATOR_HISTORY_MODIFIED_INSERTION, changeModifiedfgColor);
+	execute(SCI_INDICSETFORE, INDICATOR_HISTORY_MODIFIED_DELETION, changeModifiedfgColor);

```

Comment:
Why do we need the following lines?
```cpp
	execute(SCI_INDICSETFORE, INDICATOR_HISTORY_MODIFIED_INSERTION, changeModifiedfgColor);
	execute(SCI_INDICSETFORE, INDICATOR_HISTORY_MODIFIED_DELETION, changeModifiedfgColor);
```

Reply:
>With the PythonScript 3 plugin, a user can run editor.setChangeHistory(7) and see markers and indicators of Change History. This makes colouring indicators to match the colouring of markers too important to ignore.

>As for indicators, I consider that code that you question should be included to complete the implementation of colouring Change History in this PR.

No one asked for my advice :-) but I would code for Notepad++ "the way Notepad++ is".
Maybe don't code for it "the way it should be".  :-)

The author already has way too many things to think about (ref. open issue count), so giving him all the contingent possibilities that can be set via PythonScript (or whatever) manipulation of Scintilla is IMO a recipe for getting your change rejected.  If YOU want additional features added to _Change History_ for your own use, just code it all in PS (I do this).

Reply:
Not to stoke the fire, but see https://github.com/vinsworldcom/nppChangedLines for a plugin implementation of this feature.

Reply:
@mpheath 
> Change History is both markers and indicators and should remain in sync with colouring which seems like a good reason. What is shown depends on the setting of SCI_SETCHANGEHISTORY and can change in an instant. With the PythonScript 3 plugin, a user can run editor.setChangeHistory(7) and see markers and indicators of Change History. This makes colouring indicators to match the colouring of markers too important to ignore.

That's the valid argument. Thank you for the explanation.